web/modules/custom/nys_metatags/nys_metatags.module

```diff
@@ -0,0 +1,165 @@
+        }
+      }
+
+      if ($attachment[1] == 'og_title' || $attachment[1] == 'twitter_cards_title') {
+        if ($type == '200002' || $type == '200003' || $type == '200004') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name . '\'s ' . $title;
+        }
+        elseif ($type == '200005') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name;
+        }
+      }
+    }
+
+    if (!empty($node->field_thumbnail[0]->target_id)) {
+      $mid = $node->field_thumbnail[0]->target_id;
+      try {
+        $media = \Drupal::entityTypeManager()->getStorage('media')
+          ->load($mid);
+      }
+      catch (\Throwable) {
+        $media = NULL;
+      }
+      $fid = $media->field_image[0]->target_id;
+      $file_alt = $media->field_image[0]->alt;
+      $file = File::load($fid);
+      $file_url_generator = \Drupal::service('file_url_generator');
+      $file_uri = $file_url_generator->generateAbsoluteString($file->getFileUri());
+      $file_type = $file->getMimeType();
+
+      foreach ($metatag_attachments['#attached']['html_head'] as $id => $attachment) {
+        if ($attachment[1] == 'twitter_cards_image' || $attachment[1] == 'og_image_0') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $file_uri;
+        }
+        if ($attachment[1] == 'og_image_alt' || $attachment[1] == 'twitter_cards_image_alt') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $file_alt;
+        }
+        if ($attachment[1] == 'og_image_type') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $file_type;
+        }
+      }
+    }
+  }
+
+  if ($node->bundle() === 'article') {
+    if (!empty($node->field_social_teaser[0]->value)) {
+      $social_teaser = $node->field_social_teaser[0]->value;
+      foreach ($metatag_attachments['#attached']['html_head'] as $id => $attachment) {
+        if ($attachment[1] == 'og_description' || $attachment[1] == 'twitter_cards_description' || $attachment[1] == 'itemprop_description' || $attachment[1] == 'description') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $social_teaser;
+        }
+      }
+    }
+    if (!empty($node->field_social_image[0]->target_id)) {
+      $mid = $node->field_social_image[0]->target_id;
+      try {
+        $media = \Drupal::entityTypeManager()->getStorage('media')
+          ->load($mid);
+      }
+      catch (\Throwable) {
+        $media = NULL;
+      }
+
+      $fid = $media->field_image[0]->target_id;
+      $file_alt = $media->field_image[0]->alt;
+      $file = File::load($fid);
+      $file_url_generator = \Drupal::service('file_url_generator');
+      $file_uri = $file_url_generator->generateAbsoluteString($file->getFileUri());
```

Comment:
As above, `generateAbsoluteString()` can throw an exception.  Wrap this in `try..catch`